From 7571538cd59ec032b3006cd6a8f8e3969152e207 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Tue, 23 Aug 2022 10:54:42 +0200
Subject: [PATCH] Revert "um: Add missing apply_returns()"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Causes error compilation

arch/um/kernel/um_arch.c:447:6: error: redefinition of ‘apply_returns’
  447 | void apply_returns(s32 *start, s32 *end)
      |      ^~~~~~~~~~~~~
arch/um/kernel/um_arch.c:443:6: note: previous definition of ‘apply_returns’ with type ‘void(s32 *, s32 *)’ {aka ‘void(int *, int *)’}
  443 | void apply_returns(s32 *start, s32 *end)
      |      ^~~~~~~~~~~~~

by duplicate entry:

https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable-rc.git/tree/arch/um/kernel/um_arch.c?h=queue/5.19&id=43bf0b053a8b8d0514d7e8541c6c6dd478c77191#n447

This reverts commit 383c0862d49d18d1b08a6d471441daaade7911c2.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 arch/um/kernel/um_arch.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/um/kernel/um_arch.c b/arch/um/kernel/um_arch.c
index f87b7729c..e0de60e50 100644
--- a/arch/um/kernel/um_arch.c
+++ b/arch/um/kernel/um_arch.c
@@ -444,10 +444,6 @@ void apply_returns(s32 *start, s32 *end)
 {
 }
 
-void apply_returns(s32 *start, s32 *end)
-{
-}
-
 void apply_alternatives(struct alt_instr *start, struct alt_instr *end)
 {
 }
-- 
2.37.1.377.g679aad9e82

